javascript-bank/account.js

```diff
@@ -0,0 +1,42 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  var deposit = new Transaction('deposit', amount);
+  if (amount && typeof amount !== 'string' && amount > 0 && Number.isInteger(amount)) {
```

Comment:
The first two comparisons aren't necessary:

---

![image](https://user-images.githubusercontent.com/16143771/155402079-e9ffca64-73d0-40cc-adce-dcb0d4d51724.png)